model/registry/test.yaml

```
@@ -0,0 +1,104 @@
+groups:
+  - id: registry.test.suite
+    prefix: test
```

```suggestion
    prefix: test.suite
```

Why not extract it to the prefix and update all attributes to have id without prefix `suite`

---

Sure, I can move `suite` to be the prefix.